downstream/modules/builder/proc-customize-ee-image.adoc

```diff
@@ -41,14 +41,17 @@ collections:
 version: 1
 
 build_arg_defaults:
-  EE_BASE_IMAGE: 'example.registry.com/my-base-ee'
+  EE_BASE_IMAGE: 'registry.redhat.io/ansible-automation-platform-22/ee-minimal-rhel8:latest'
 
 dependencies:
   galaxy: requirements.yml
 ----
 ====
 +
-NOTE: Since this example uses the community version of `kubernetes.core` and not a certified collection from {HubName}, we do not need to create an `ansible.cfg` nor reference that in our definition file.
+[NOTE]
+====
+Since this example uses the community version of `kubernetes.core` and not a certified collection from {HubName}, we do not need to create an `ansible.cfg` nor reference that in our definition file.
```

Comment:
```suggestion
Since this example uses the community version of `kubernetes.core` and not a certified collection from {HubName}, we do not need to create an `ansible.cfg` file or reference that in our definition file.
```